src/utils/ContractUtils.ts

```diff
@@ -149,8 +158,10 @@ export async function sendTx(
   overrides.gasLimit = estGas.add(20000)
   try {
     const trxReceipt = await functionToSend(...args, overrides)
+    const receipt = await trxReceipt.wait()
```

Comment:
`'receipt' is assigned a value but never used`

```suggestion
   await trxReceipt.wait()
```